compose/compose-file/index.md

```diff
@@ -882,6 +882,81 @@ port (a random host port will be chosen).
      - "127.0.0.1:5000-5010:5000-5010"
      - "6060:6060/udp"
 
+### secrets
+
+After you
+[define or reference secrets in the top-level `secrets` configuration](#secrets-configuration-reference),
+grant access to secrets on a per-service basis using the per-service `secrets`
+configuration. Two different syntax variants are supported.
+
+#### Short syntax
+
+The short syntax variant only specifies the secret name. This grants the
+container access to the secret and mounts it at `/run/secrets/<secret_name>`
+within the container. The source name and destination mountpoint are both set
+to the secret name.
+
+> **Warning**: Due to a bug in Docker 1.13.1, using the short syntax currently
+> mounts the secret with permissions `000`, which means the secret is unreadable
+> if you define it using the short syntax. The workaround is to use the long
```

Comment:
Being accurate about the scope of a bug is hardly encouraging anyone to run containers as root.

The bug is that non-root users can not read the file. Claiming otherwise is misleading and more likely to prevent people from adopting secrets. If they need them as a non-root user, we already have the workaround documented (they can set the mode with the long form), which is much easier than running as root.